storm/big/Ops.cpp

```diff
@@ -393,7 +393,8 @@ void Sub(BigBuffer& a, const BigBuffer& b, const BigBuffer& c) {
 
     a.SetCount(i);
 
-    STORM_ASSERT(!borrow);
+    // This assert does not exist in retail WoW or Starcraft.
+    //STORM_ASSERT(!borrow);
```

Comment:
Hmm. This assertion is present in assertion-enabled builds of WoW. Retail builds generally have assertions disabled, which would explain why you perhaps don't see it in a stable build of WoW or SC.

I'd prefer we keep this and discuss whether Whoa's assert macros have the right behavior. It should be possible to disable assertions such that a build without assertions doesn't prevent the desired bug.